srcpkgs/ccache/template

```diff
@@ -15,11 +13,31 @@ changelog="https://ccache.dev/releasenotes.html"
 distfiles="https://github.com/ccache/ccache/releases/download/v${version}/${pkgname}-${version}.tar.xz"
 checksum=ac1b82fe0a5e39905945c4d68fcb24bd0f32344869faf647a1b8d31e544dcb88
 
+if [ -z "$CHROOT_READY" ]; then
+	hostmakedepends="cmake-bootstrap"
+	# tests require ctest, not available in cmake-bootstrap
+	configure_args+=" -DENABLE_TESTING=OFF"
+else
+	hostmakedepends="asciidoc cmake perl"
+
+	if [ -n "$XBPS_CHECK_PKGS" ]; then
+		configure_args+=" -DENABLE_TESTING=ON"
+	else
+		configure_args+=" -DENABLE_TESTING=OFF"
+	fi
+fi
```

Comment:
I would try to avoid nested `if .. then ..fi`
I guess something like:
```sh
if [ -z "$CHROOT_READY" ]; then
	hostmakedepends="cmake-bootstrap"
	unset XBPS_CHECK_PKGS
else
	hostmakedepends="asciidoc cmake perl"
fi

if [ -n "$XBPS_CHECK_PKGS" ]; then
	configure_args+=" -DENABLE_TESTING=ON"
else
	configure_args+=" -DENABLE_TESTING=OFF"
fi
```

The `unset` is also used in `binutils`.

I'll try a full bootstrap later.

Reply:
please, just don't

messing with xbps-src variables from template is terrible, keep the nested ifs

Reply:
I feel like it is a recipe for breaking something changing XBPS_*